packages/rake-db/src/migration/migrationList.ts

```diff
@@ -0,0 +1,31 @@
+import { ColumnTypesBase } from 'orchid-core';
+import { DefaultColumnTypes } from 'pqb';
+import { RakeDbConfig, getMigrations } from 'src/common';
+
+const getFileNameFromPath = (path: string) => {
+  const pathArray = path.split('/');
+  // pick the file name from path & remove the time stamp.
+  const fileName = pathArray[pathArray.length - 1].slice(15);
+  return fileName;
+};
+
+const parseMigrationListRow = (path: string, direction: 'up' | 'down') =>
+  `${direction} ${getFileNameFromPath(path)} ${path}`;
+
+export const migrationList = async <
+  C extends ColumnTypesBase = DefaultColumnTypes,
+>(
+  config: RakeDbConfig<C>,
+) => {
+  // get list of down migrations & log them.
+  const downMigrationsList = await getMigrations(config, false);
+  downMigrationsList.forEach((migrationItem) => {
+    config.logger?.log(parseMigrationListRow(migrationItem.path, 'down'));
+  });
+
+  // get list of up migrations & log them.
+  const upMigrationsList = await getMigrations(config, true);
+  upMigrationsList.forEach((migrationItem) => {
+    config.logger?.log(parseMigrationListRow(migrationItem.path, 'up'));
+  });
```

Comment:
Migrations should be outputted in a historical order.

Reply:
sorted first by [direction, date] or [date, direction]?

Reply:
`getMigrations` returns them in proper order: from old to new.
```
await getMigrations(config, true);
```

No need to sort by "direction". `up` or `down` is a status - was it migrated or not.